CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Contributing
+
+This document provides guidelines for contributing to samples.
+
+## Dependencies
+
+The following dependencies must be installed on the development system:
+
+- [Docker Engine][docker-engine]
+- [Google Cloud SDK][google-cloud-sdk]
+- [make]
+
+## Integration Testing
+
+Integration tests are used to verify that the samples are actuatable 
+by Terraform. Tests are dynamically discovered based on directories in
+repo root and executed using the [blueprint-test](https://pkg.go.dev/github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test) framework.
+
+### Test Environment
+
+The easiest way to test the samples are in isolated test projects. The
+setup for projects are defined in [test/setup](./test/setup/)
+directory.
+
+To use this setup, you need to authenticate gcloud with an identity that has
+Project Creator access on a folder and the Billing Account User role on a billing account.
+
+Set the following environment variables.
+```
+export TF_VAR_org_id="your_org_id"
+export TF_VAR_folder_id="your_folder_id"
+export TF_VAR_billing_account="your_billing_account_id"
+```
+
+With these settings in place, you can prepare a test project using Docker:
+
+```
+cd terraform-docs-samples
+make docker_test_prepare
```

Comment:
For me, this runs successfully in Cloud Shell.

On my Mac, I get the following error:

```
$ make docker_test_prepare
docker run --rm -it \
		-e SERVICE_ACCOUNT_JSON \
		-e TF_VAR_org_id \
		-e TF_VAR_folder_id \
		-e TF_VAR_billing_account \
		-v "/Users/betsyl/terraform/terraform-docs-samples":/workspace \
		gcr.io/cloud-foundation-cicd/cft/developer-tools:1 \
		/usr/local/bin/execute_with_credentials.sh prepare_environment
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
ERROR: (gcloud.auth.activate-service-account) Missing required argument [ACCOUNT]: An account is required when using .p12 keys
ERROR: (gcloud.auth.activate-service-account) Missing required argument [ACCOUNT]: An account is required when using .p12 keys
make: *** [docker_test_prepare] Error 1
```